### PR TITLE
14384 - Fix for routing_slip_task.py

### PR DESCRIPTION
**Pull request by seeker25**

### CODE REVIEW

**Review by Jxio — APPROVED** [review]

**Comment by seeker25** [comment]
Not really sure why linting is failing - probably unrelated to us.

File: jobs/payment-jobs/requirements/prod.txt [hunk 1]
```diff
@@ -17,5 +17,5 @@ dataclass_wizard
 launchdarkly-server-sdk
 
 -e git+https://github.com/bcgov/sbc-common-components.git#egg=sbc-common-components&subdirectory=python
--e git+https://github.com/bcgov/sbc-pay.git@main#egg=pay_api&subdirectory=pay-api
+-e git+https://github.com/seeker25/sbc-pay.git@14384#egg=pay_api&subdirectory=pay-api
```
Comment (seeker25) [inline jobs/payment-jobs/requirements/prod.txt:20]: Will put this back in another PR after merged.
